app/src/main/java/ch/blinkenlights/android/medialibrary/MediaScanner.java

```diff
@@ -500,19 +500,19 @@ private boolean rpcInspectFile(File file) {
 
 			// Get tags which always must be set
 			String title = tags.getFirst(MediaMetadataExtractor.TITLE);
-			if (title == null)
+			if (title == null || title.trim().isEmpty())
```

Comment:
Could you create a static function for these checks?

Something like
```
private static boolean isUnset(String s) {
   return s == null || s.trim().isEmpty();
}
```

Reply:
Where would you want this function to be placed? I'm not too familiar with the code.